blog/docs/articles/using-sideloaded-images-with-kind.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Using Sideloaded Images with Kind
+linkTitle: Using-Sideloaded -Images-with-Kind
+author: "Ed Jung"
+date: 2022-05-26
+description: How to use sideloaded images with Knative on Kind
+type: "blog"
+---
+
+Once you've setup a kind Knative cluster with the [`quickstart` plugin](https://github.com/knative-sandbox/kn-plugin-quickstart), you may want to try it with images from your developer environment, without an image registry.
```

Comment:
```suggestion
Once you have set up a kind Knative cluster by using the [`quickstart` plugin](../install/quickstart-install.md/), you can use images from your developer environment without an image registry.
```

Suggestions for readability, also please update the link instead to link to https://knative.dev/docs/install/quickstart-install/ by linking within the docs (see https://github.com/knative/docs/blob/main/contribute-to-docs/style-guide/style-and-formatting.md#links)